openstudiocore/src/airflow/contam/ForwardTranslator.cpp

```diff
@@ -434,10 +434,18 @@ boost::optional<contam::IndexModel> ForwardTranslator::translateModel(model::Mod
   m_logSink.setThreadId(QThread::currentThread());
   m_logSink.resetStringStream();
 
-  m_prjModel.read(std::string(":/templates/template.prj"));
-  if (!m_prjModel.valid()) {
-    return boost::none;
+  {
+    QFile f(":/templates/template.prj");
+    f.open(QFile::ReadOnly);
+    QString s = f.readAll();
+    Reader r(&s);
```

Comment:
Nope, Reader was there all along, but Reader no longer deals in terms of QFiles.